resource/require.go

```diff
@@ -0,0 +1,25 @@
+package resource
+
+import (
+	"testing"
+	"os"
+)
+
+const (
+	UnitTest = "ALMIGHTY_RESOURCE_UNIT_TEST"
+	Database = "ALMIGHTY_RESOURCE_DATABASE"
+	StSkipReason = "Skipping unit test because environment variable %s is no set."
```

Comment:
Sorry, I was unclear. I meant "Skipping test....." as oppose to "Skipping unit test....". This might not be a "unit test", right? :)